src/index.html

```diff
@@ -1,19 +1,323 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
-          user-scalable=no,
-          initial-scale=1.0,
-          maximum-scale=1.0,
-          minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Document</title>
+        <link rel="stylesheet" href="./style.css">
+        <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">
+    </head>
+    <body>
+        <header>
+            <a href="#" class="imgLogo">
+                <img src="images/logo.png" alt="logo">
+            </a>
+            <nav>
+                <a href="#" class="is-active">Apple</a>
+                <a href="#">Samsung</a>
+                <a href="#">Смарфоны и телефоны</a>
+                <a href="#" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+                <a href="#">Гаджеты</a>
+                <a href="#">Планшеты</a>
+                <a href="#">ФОТО</a>
+                <a href="#">ВИДЕО</a>
+            </nav>
+        </header>
+        <main>
+            <div class="card" data-qa="card">
+                <div class="cardImg">
+                    <img src="images/imac.jpeg" alt="imac">
+                </div>
+                <div class="cardTitle">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+                <div class="itemId">Код товара: 195434
+        </div>
+                <div class="feedback">
+                    <div class="stars">
+                        <div>
+                            <img src="images/star-active.svg" alt="star-active">
+                        </div>
+                        <div>
+                            <img src="images/star-active.svg" alt="star-active">
+                        </div>
+                        <div>
+                            <img src="images/star-active.svg" alt="star-active">
+                        </div>
+                        <div>
+                            <img src="images/star-active.svg" alt="star-active">
+                        </div>
+                        <div>
+                            <img src="images/star.svg" alt="star">
+                        </div>
+                    </div>
+                    <div class="feedbackTitle">Отзывов: 5
+                </div>
+                </div>
+                <div class="priceWrapper">
```

Comment:
Do we need 2 `div` in `div`? Too many `div`.
Maybe better to use `p` or something like that?